common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -516,4 +516,13 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  val NewNavEnabled = Switch(
+    SwitchGroup.Feature,
+    "new-nav-enabled",
+    "If this is switched on then the new navigation will be enabled",
+    owners = Seq(Owner.withGithub("GHaberis")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
Do we want to stick a date on this to ~annoy the hell out of us~ remind us to tidy up this code?

Reply:
Yes definitely! I would probably have it expire a week from Monday?